
# The Global Soil Mycobiome consortium

## About

Fungi are an incredibly diverse group of organisms that play a crucial role in providing a wide range of ecosystem services. Despite their importance, fungi have received far less attention compared to animals and plants. As a result, we have limited knowledge about the distribution patterns and conservation needs of fungi. 
The Global Soil Mycobiome Consortium (GSMc), led by researchers in the Mycology and Microbiology Center (MMC), University of Tartu (link to MMC) was initiated in 2015 to fill this gap in our understanding. Building on the success of a previous global soil fungal survey (link), the GSMc has significantly expanded the geographical and habitat coverage of soil fungi.
The current dataset includes 3200 composite samples (comprising a total of 127 263 sub-samples) collected from sites located across 108 different countries. (ADD INTERACTIVE MAP HERE) As a result, this dataset is the largest and most comprehensive collection of soil fungal distribution data obtained following standardized procedures. 
The GSMc dataset is readily available to the public over the PlutoF repository (ADD LINKS), allowing researches from around the world to access this invaluable resource.
An additional mission of the GSMc is to reward contributing researchers with scientific currency – co-authorships.

The dataset provided by the Global Soil Mycobiome consortium (GSMc) represents a significant advancement over previous public datasets and databases in several important ways: 
(1)	The geographical breadth and number of individual samples (> 125,000) are by far the greatest to date;
(2)	The molecular barcode covers both the entire ITS region and the V9 variable region of the 18S rRNA gene. This combination provides a much greater taxonomic resolution than the ITS1 and ITS2 subregions alone;
(3)	All samples have been collected and processed following the same protocol;
(4)	Molecular analyses such as PCR and library preparation for high-throughput sequencing have been performed in a single laboratory (Mycology and Microbiology Center, University of Tartu);
(5)	To ensure accuracy, the data has been carefully quality-filtered by checking for potential chimeras and contaminants manually for each sequencing run.
(6)	The dataset is equipped with up-to-date taxonomic and functional annotations and metadata, allowing users to reinterpret and analyze the data in new ways. 
(7)	The dataset is freely available to all researchers in a ready-to-use format to encourage incorporation of soil fungi into macroecological analyses and boost understanding of the diversity and distribution of fungi. 

A new release including a short data paper is planned for the year 2024. The progress and early access can be followed via Twitter, @tedersoo 


## Funding

Funding of the fieldwork and shipping comes from the GSMc partners. Molecular work is performed by students in the MMC lab and is funded by the following grants: Estonian Science Foundation (grants PRG632, rein), MOBERC, Norway-Baltic … EMP442, Novo Nordisk Foundation (SilvaNova) and King Saud University (DSFP-2021-2023).


## Publications
Core publications

Tedersoo, L., Mikryukov, V., Anslan, S. et al. The Global Soil Mycobiome consortium dataset for boosting fungal diversity research. Fungal Diversity 111, 573–588 (2021). https://doi.org/10.1007/s13225-021-00493-7

Tedersoo, L. et al., 2022. Global patterns in endemicity and vulnerability of soil fungi. https://doi.org/10.1111/gcb.16398 

Papers using mostly GSMc DNA samples or data

Tedersoo L, Anslan S, Bahram M, Drenkhan R, Pritsch K, Buegger F, Padari A, Hagh-Doust N, Mikryukov V, Kõljalg U, Abarenkov K. 2020. Regional-scale in-depth analysis of soil fungal diversity reveals strong pH and plant species effects in Northern Europe. Frontiers in Microbiology 11:1953.

Davison J, Moora M, Semchenko M, Adenan SB, Ahmed T, Akhmetzhanova AA, Alatalo JM, Al‐Quraishy S, Andriyanova E, Anslan S, Bahram M, Tedersoo L, Zobel M, Öpik M et al. 2021. Temperature and pH define the realised niche space of arbuscular mycorrhizal fungi. New Phytologist 231: 763-776.

Vasar M, Davison J, Sepp S-K, Mucina L, Oja J, Al-Quraishy S, Anslan S, Bahram M, Bueno CG, Cantero JJ, Decocq G, Fraser L, Hiiesalu I, Hozzein WN, Koorem K, Meng Y, Moora M, Onipchenko V, Öpik M, Tedersoo L, Zobel M. 2022. Global soil microbiomes: A new frontline of biome-ecology research. Global Ecology and Biogeography 31: 1120-1132.

Sepp SK, Vasar M, Davison J, Oja J, Anslan S, Al-Quraishy S, Bahram M, Bueno CG, Cantero JJ, Fabiano EC, Decocq G. et al. 2023. Global diversity and distribution of nitrogen-fixing bacteria in the soil. Frontiers in Plant Science 14:1100235.

Vasar M, Davison J, Moora M, Sepp SK, Anslan S, Al-Quraishy S, Bahram M, Bueno CG, Cantero JJ, Fabiano EC, Decocq G. et al. 2023. Metabarcoding of soil environmental DNA to estimate plant diversity globally. Frontiers in Plant Science 14:1106617.

## Collaborators


[Link text](https://website-name.com)
[Link text](publications.md)



....
